app/src/main/java/me/ivg2/foodapp/HomeActivity.java

```diff
@@ -1,13 +1,51 @@
 package me.ivg2.foodapp;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.view.MenuItem;
 
 public class HomeActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
+
+        final FragmentManager fragmentManager = getSupportFragmentManager();
+
+        final Fragment recipeFragment = new RecipeFragment();
+        final Fragment addFoodFragment = new AddFoodFragment();
+        final Fragment fridgeFragment = new FridgeFragment();
+
+        FragmentTransaction fragmentTransaction1 = fragmentManager.beginTransaction();
+        fragmentTransaction1.replace(R.id.homeFragment, recipeFragment).commit();
+
+        BottomNavigationView bottomNavigationView = findViewById(R.id.bottom_navigation);
+
+        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                //currently I only need two cases - home timeline and creating a post
+                switch (item.getItemId()) {
```

Comment:
Seems like you can pull out the 

` FragmentTransaction fragmentTransaction1 = fragmentManager.beginTransaction();`

From the switch case and just do  inline transactions like I described above.